caas/Dockerfile

```diff
@@ -5,6 +5,9 @@ FROM $BASE_IMAGE
 RUN useradd --system -M syslog
 RUN usermod -s /usr/sbin/nologin syslog
 
+# Add a juju user to run the daemon.
+RUN useradd -u 1010 juju
```

Comment:
We need to setup a home for juju, add juju to the sudoers etc. Charms should still be able to install apt packages etc, so they will need sudo. Eventually this would probably become a charm "assume" (so that a charm has to opt into sudo).

Reply:
I think just to add to this we probably want a nologin shell on this user as well and I am not sure we would need a home directory? Should we look at giving the group created a well known GID as well if we are fixing these at good locations?